Assets/Scripts/UI/TilingCowbots.cs

```diff
@@ -0,0 +1,42 @@
+using UnityEngine;
+using UnityEngine.UI;
+
+public class TilingCowbots : MonoBehaviour
+{
+    [SerializeField] private RawImage cowboys;
+    [SerializeField] private bool rotateCowbots = false;
+    [SerializeField, Range(0, 2)] private float speedMultiplier = 0.5f; 
+    private Vector2 cowbotVelocity;
+
+    void Start()
+    {
+        if (rotateCowbots)
```

Comment:
Please make this the default for the loading screen at least.

Reply:
Fixed